python/lsst/ts/observatory/control/maintel/comcam.py

```diff
@@ -240,7 +241,43 @@ async def setup_filter(self, filter):
         if filter is None:
             return
         else:
-            raise NotImplementedError("set_filter not available yet.")
+            self.rem.cccamera.cmd_setFilter.set(name=filter)
```

Comment:
Any reason for not using `set_start` instead of `set` and then `start`?

Also, may be a good idea to provide input check. Since you have `get_available_filters`, you could make sure the filter passed is valid and raise a `RuntimeError` if it is not. You may want to skip the check if you cannot determine the filter (logging a warning, for instance).

Reply:
Can use `set_start`.

Reply:
Not sure what you mean by "skip the check if you cannot determine the filter". Is that in the `case filter = None`? If that's true then logging a warning for skipping the check seems odd.